test/end2end/model_special_cases_test.dart

```diff
@@ -203,6 +203,31 @@ void main() {
         expect(referenceLookup(constructorTearoffs, 'Ft.new'),
             equals(MatchingLinkResult(Fnew)));
       });
+
+      test('we can use (ignored) type parameters in references', () {
+        expect(referenceLookup(E, 'D<String>.new'),
```

Comment:
A type argument can also be a prefixed identifier, like `<core.String>`.

By all means, does not have to be addressed in this CL, but maybe a TODO.

Reply:
Yes.  Didn't add a test for that, but should be supported by the code here as well.